minidump-processor/src/op_analysis.rs

```diff
@@ -18,6 +18,7 @@
 
 use minidump::{MinidumpContext, MinidumpRawContext, UnifiedMemory};
 use std::collections::BTreeSet;
+use yaxpeax_x86::amd64::Instruction;
```

Comment:
This needs to be gated behind `#[cfg(feature = "disasm_amd64")]` like the other bits in this module, since `yaxpeax-x86` is an optional dependency tied to that feature.